### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by techmoves**
in this repo idid this :👍 

- [x]  Created a new directory called components, Inside components, created a new Calculator.js file.

- [x] the componets matches the design

- [x] No linters error

- [x] Used functional components

- [x] Git flow used

### CODE REVIEW

File: src/componets/calculator.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*eslint-disable */
```
Comment (wayungi) [inline src/componets/calculator.js]: - [ ] Please do not disable linters. Linters enforce good coding standards. 👍🏾

File: src/componets/calculator.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import "./calculator.css"; 
+
+const Calculator = ({ symbol, Color }) => {
+  return (
+    <div className="calc-wrapper"style={{ backgroundColor:Color}}>
+      {symbol}
+    </div>
+  );
+};
```
Comment (wayungi) [inline src/componets/calculator.js]: - [ ] Good work building the calculator component. It is a project requirement that the Calculator component should have at least 1 child component. This means you need to import another component and use it in the calculator class. 🤣

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @techmoves , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ]  Correct Gitflow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @techmoves , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great requirement implementation
✅ ...  Green linters

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**